CHANGELOG.md

```diff
@@ -1,9 +1,22 @@
 ## master / unreleased
 
-* [CHANGE] In table-manager, default DynamoDB capacity was reduced from 3,000 units to 1,000 units. We recommend you do not run with the defaults: find out what figures are needed for your environment and set that via `-dynamodb.periodic-table.write-throughput` and `-dynamodb.chunk-table.write-throughput`.
+## 0.3.0 / 2019-10-07
```

Comment:
Shold be `-rc.0` https://github.com/cortexproject/cortex/pull/1623/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR3